browser/.beforeprettier

```diff
@@ -21,7 +21,7 @@
 /src/control/Control.LanguageDialog.js
 /src/control/Control.Layers.js
 /src/control/Control.LokDialog.js
-/src/control/Control.Mention.js
+/src/control/Control.Mention.ts
```

Comment:
can we remove that so we will get new formatting?
then please do: `make prettier-write` to apply unified formatting